requirements/prod.txt

```diff
@@ -274,22 +259,17 @@ feedparser==6.0.8 \
     --hash=sha256:5ce0410a05ab248c8c7cfca3a0ea2203968ee9ff4486067379af4827a59f9661
 filtercascade==0.4.1 \
     --hash=sha256:1db4c2f1f628e3e399a6f3437738bdb304eb11521bf37684b56addaa5576a89f
-# funcsigs is required by mock
-funcsigs==1.0.2 \
-    --hash=sha256:330cc27ccbf7f1e992e69fef78261dc7c6569012cf397db8d3de0234e6c937ca \
-    --hash=sha256:a7bb0f2cf3a3fd1ab2732cb49eba4252c2af4240442415b4abce3b87022a8f50
 geoip2==4.3.0 \
     --hash=sha256:7ad10942e9fd6c54bc850d8311618f04dacd3fff5b55ba48b7ad83502fc884bd \
     --hash=sha256:599914784cea08b50fb50c22ed6a59143b5ff2d027ba782d2d5b6f3668293821
-# html5lib is required by rdflib
```

Comment:
Did you intend to just remove this comment and not replace it with anything?

Reply:
Yes: it's not true, we directly use `html5lib` (and don't use `rdflib` anymore).